devTools/phpstan-src.neon

```diff
@@ -1,8 +1,7 @@
 parameters:
-    excludes_analyse:
-        - %currentWorkingDirectory%/tests/Fixtures/*
-
     ignoreErrors:
         - '#Access to an undefined property PhpParser\\Node(.*?)#'
         - '#Call to an undefined method PhpParser\\Node(.*?)#'
         - '#Cannot access property \$name on string.#'
+        - '#Accessing property \$expr on possibly null value of type PhpParser\\Node(.*?)#'
+        - '#expects PhpParser\\Node\\Identifier, string given.#'
```

Comment:
This error is most likely caused due to phpstan still using v3 of the parser, which causes it to compare it against the class they use instead of our version. (probably)